### PR TITLE
Add developer hub content to README

### PR DESCRIPTION
**Pull request by triblondon**
As suggested by @acfoltzer, using a CODEOWNERS file to flag the README as a thing of interest to devrel.  I'd be happy to drop the entry there for fastly.toml if we prefer since it's pretty structured.

### CODE REVIEW

File: README.md
Comment (phamann) [inline README.md:5]: This URL currently 404's, i'm assuming as the content hasn't been launched yet? Should we not merge this PR until said content is live?

File: README.md
Comment (phamann) [inline README.md:14]: The link to our `SECURITY.md` in the readme is a requirement from our security team for all open-source projects. Please add back in at the bottom of the page.

File: README.md
Comment (phamann) [inline README.md:33]: I'd consider add an example CLI command to add a backend below this. Such as:
```suggestion
The starter uses two backends, so if you want to, go ahead and create two backends using the CLI and then modify both names here. You should now have a Fastly service running on Compute@Edge that can talk to your backends, and generate synthetic responses at the edge.

You can create a backend using the CLI:
```term
$ fastly backed create --name example --address www.example.com
```
```

**Review by phamann — CHANGES_REQUESTED** [review]
LGTM, please add back SECURITY.md reference.

File: README.md
Comment (triblondon) [reply README.md:33]: Better to link to canonical sources where they exist - and we actually already have a link to the relevant CLI command page in a preceding para.

File: README.md
Comment (triblondon) [reply README.md:5]: This is a chicken/egg problem.  We could either not care too much and just try to co-ordinate the merges to be close together, or we could split it up so that this line is added to the readme in a subsequent PR.

Comment (phamann) [reply README.md:5]: I quick co-ordination of merges should be sufficient :+1:

**Review by phamann — APPROVED** [review]
👍 LGTM

**Pull request merged**
